### DISCUSSION TITLE
custom action for resource, e.g. "refresh data" button at top of resource page?

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#6058] [Help]
Maybe I missed this in the documentation...

(How) can I add a custom action to a resource?
So: a button that appears at the top of a resource table view, similar to "create new <model>"

E.g. "refresh data" action where I can call a function to update the models based on external API.

**Comment by danharrin** [comment]
You can add an action to the List page - https://filamentphp.com/docs/2.x/admin/pages/actions
